### ISSUE TITLE
Dynamic hostname

### ISSUE THREAD

**Issue by cknowles** [bsm/datadog-notifications#7]
In an environment like Kubernetes, it seems recommended to not include the `host` tag in custom metrics as this gets picked up automatically by the agent when it's configured correctly. Currently [the code](https://github.com/bsm/datadog-notifications/blob/004377de7d4ef6feec5d1b11e99ca96def99d26f/lib/datadog/notifications/config.rb#L23) will default to the pod name which in turn means the host tags are not picked up inside datadog.

It's possible to override it using the [Downward API](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/):
```
- name: INSTRUMENTATION_HOSTNAME
  valueFrom:
    fieldRef:
      fieldPath: spec.nodeName
```

That has to be done on every pod sending custom metrics but maybe we could add a flag or similar to not send the `host` tag?

**Comment by dim** [comment]
How about setting the `hostname` to `false`? In that case, it could skip the tag altogether. Would you be able to submit a PR? Cheers

**Comment by cknowles** [comment]
I think that sounds reasonable. Can submit a PR when we get back to the related work.

**Comment by dim** [comment]
@c-knowles any update? I will close this one in the meantime

**Comment by cknowles** [comment]
Ah, this is resolved in https://github.com/bsm/datadog-notifications/pull/10 now I think which is merged. Is there a release with that?

**Comment by dim** [comment]
Yes, released 0.5.1 today

**Comment by mxmCherry** [comment]
@dim close this now?
